WorkunitExec.ecl

```diff
@@ -522,6 +571,84 @@ EXPORT WorkunitExec := MODULE
             );
 
         RETURN queryResults(rProtected);
-    END;
+    END;  
+
+    /**
+     * Record structure used to represent the results of a published
+     * workunit.
+     */
+     EXPORT PublishResultsLayout := RECORD
```

Comment:
Formatting: Please vertically-align the XPATH field attribute (see RunResultsLayout as an example).